calibration/CTDBPE/CGINS-CTDBPE-50067__20210112.csv

```diff
@@ -0,0 +1,23 @@
+serial,name,value,notes
+16-50067,CC_a0,1.223382e-03,source_file 3305-00102-00267-A_SN_16-50067_QCT_Results_CTDBP-E.cap
+16-50067,CC_a1,2.816546e-04,
+16-50067,CC_a2,-1.880341e-06,
+16-50067,CC_a3,2.145008e-07,
+16-50067,CC_cpcor,-9.570000e-08,
+16-50067,CC_ctcor,3.250000e-06,
+16-50067,CC_g,-1.010556e+00,
+16-50067,CC_h,1.611004e-01,
+16-50067,CC_i,-2.385492e-04,
+16-50067,CC_j,4.514406e-05,
+16-50067,CC_pa0,1.575270e+00,
+16-50067,CC_pa1,2.621756e-03,
+16-50067,CC_pa2,1.863044e-11,
+16-50067,CC_ptca0,5.243560e+05,
+16-50067,CC_ptca1,4.326694e+01,
+16-50067,CC_ptca2,-7.231173e-01,
+16-50067,CC_ptcb0,2.499000e+01,
+16-50067,CC_ptcb1,-0.000000e+00,
```

Comment:
This value is 0.000000e-41 in the cal capture. Don't think that matters though. Looks good